Formula/libgr.rb

```diff
@@ -12,9 +12,16 @@ class Libgr < Formula
 
   depends_on :xcode => :build
   depends_on "cairo"
+  depends_on "ffmpeg"
```

Comment:
Are `qt` and `ffmpeg` always necessary? They are quite heavy dependencies…

Reply:
`qt`: Yes. Because https://github.com/sciapp/gr/blob/master/lib/gks/Makefile#L48-L50 always includes `gksqt`. Should we send a pull request to GR framework to make `gksqt` optional and retry the libgr formula upgrade?

`ffmpeg`: No. We can use GR framework without FFmpeg. But we can't use video output feature. I've removed this.